src/core_plugins/i18n/server/i18n/i18n.js

```diff
@@ -0,0 +1,78 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+
+const readFile = Promise.promisify(fs.readFile);
+
+const TRANSLATION_FILE_EXTENSION = 'json';
+
+let registeredTranslations = {};
```

Comment:
I don't see this variable getting reassigned anywhere, why do you need to use let?